Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -251,49 +251,71 @@ function New-LDAPIdentitySource {
 
 <#
     .Synopsis
-     Download the certificate from a domain controller
+     Download certificates from domain controllers and save them to local files
 #>
-function Get-CertificateFromDomainController {
+function Get-CertificateFromDomainContollerToLocalFile {
```

Comment:
Let's not use Domain Controller - yes, it's a most common type of LDAP customers have, but that's not the defining characteristic of it - we don't if it's a DC in the slightest, it just needs to be TLS-protected LDAP server.
It's also mispelled.